service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/MessageJudgeControllerSubmittedTest.java

```diff
@@ -112,7 +115,8 @@ void shouldNotifyJudicialMessageRecipientWhenJudicialMessageReplyAdded() throws
 
         Map<String, Object> expectedData = Map.of(
             "respondentLastName", "Davidson",
-            "caseUrl", "http://fake-url/cases/case-details/12345#JudicialMessagesTab",
+            "caseUrl", String.format("http://fake-url/cases/case-details/%s#%s", CASE_REFERENCE.toString(),
```

Comment:
probs meant to change this

Reply:
luckily i found it just now whilst fixing the conflict anyway!